examples/mysql-wordpress-pd/README.md

```diff
@@ -34,301 +34,231 @@ Documentation for other releases can be found at
+store their data. We will use a Persistent Volume Claim to claim an
+available persistent volume. This example covers HostPath and
+GCEPersistentDisk volumes. Choose one of the two, or see
+[Types of Persistent Volumes](../../docs/user-guide/persistent-volumes.md#types-of-persistent-volumes)
+for more options.
+
+### GCE Persistent Disk
+
+Please see the
+[GCE getting started guide](../../docs/getting-started-guides/gce.md)
+for full details and other options for starting a cluster.
+
+Create two persistent disks. You will need to create the disks in the
+same [GCE zone](https://cloud.google.com/compute/docs/zones) as the
+Kubernetes cluster. The default setup script will create the cluster
+in the `us-central1-b` zone, as seen in the
+[config-default.sh](../../cluster/gce/config-default.sh) file. Replace
+`<zone>` below with the appropriate zone. The names `wordpress-1` and
+`wordpress-2` must match the `pdName` fields we have specified in
+[gce-volumes.yaml](gce-volumes.yaml).
+
+```shell
+gcloud compute disks create --size=20GB --zone=<zone> wordpress-1
+gcloud compute disks create --size=20GB --zone=<zone> wordpress-2
 ```
 
-Please see the [GCE getting started guide](../../docs/getting-started-guides/gce.md) for full details and other options for starting a cluster.
-
-## Create two persistent disks
-
-For this WordPress installation, we're going to configure our Kubernetes [pods](../../docs/user-guide/pods.md) to use [persistent disks](https://cloud.google.com/compute/docs/disks). This means that we can preserve installation state across pod shutdown and re-startup.
-
-You will need to create the disks in the same [GCE zone](https://cloud.google.com/compute/docs/zones) as the Kubernetes cluster. The default setup script will create the cluster in the `us-central1-b` zone, as seen in the [config-default.sh](../../cluster/gce/config-default.sh) file. Replace `$ZONE` below with the appropriate zone.
+Create the persistent volume objects in Kubernetes for those disks:
 
-We will create two disks: one for the mysql pod, and one for the wordpress pod. In this example, we create 20GB disks, which will be sufficient for this demo. Feel free to change the size to align with your needs, as wordpress requirements can vary. Also, keep in mind that [disk performance scales with size](https://cloud.google.com/compute/docs/disks/#comparison_of_disk_types).
-
-First create the mysql disk.
-
-```sh
-gcloud compute disks create --size=20GB --zone=$ZONE mysql-disk
+```shell
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/gce-volumes.yaml
 ```
 
-Then create the wordpress disk.
+### Host Path
 
-```sh
-gcloud compute disks create --size=20GB --zone=$ZONE wordpress-disk
-```
+Host paths are volumes mapped to directories on the host. These should
+be used for testing or single-node clusters only, as the data will not
+be moved between nodes if the pod is recreated on a new node. Create
```

Comment:
The DeamonSet for Cassandra is pretty cool--how difficult is it to set up a distributed MySQL deployment?

> These should be used for testing or single-node clusters only, as the data will not be moved between nodes if the pod is recreated on a new node. 

I would emphasis  this more (italics maybe), maybe clarify what exactly happens as a result (MySQL pod gets moved to a new node, data remains on old node, apparent data loss). And maybe consider using a nodeName or nodeSelector to ensure the pod always lands on the same machine? Still bad if the node goes down, but at least data won't disappear due to pod getting rescheduled.